### ISSUE TITLE
Strange changes/missing devices after restart.

### ISSUE THREAD

**Issue by Zebble** [home-assistant/hassio-addons#860]
I have Hass.io installed in an x86 VM, along with this deCONZ add-on.

Everything works well, as long as I don't restart the deCONZ add-on.

After restart, very often devices (usually lights) will be missing from the web app, or renamed and groups will very often be re-renamed along with new groups being added that correspond with a dimmer or motion device.

In the VNC console, the missing devices in the web app are *not* missing in the zigbee map.

Sounds like a database persistence problem, but I'm not sure where to start!

**Comment by pvizeli** [comment]
https://github.com/dresden-elektronik/deconz-rest-plugin

**Comment by pvizeli** [comment]
You can also jump into the container and look if there are files outside `/data`

**Comment by Zebble** [comment]
Thanks @pvizeli .  Everything looks good.  All data appears to be in /data, which is a persistent volume.

However, I did find this on the deCONZ REST API plugin github:

https://github.com/dresden-elektronik/deconz-rest-plugin/issues/1697

Appears to be that any device/group names with a single quote in them are not being saved.  They save in the interface, but don't actually end up in the database.

Changed them to not include the quote, and all working as expected.  Had this problem for months to the point where I flipped to ZHA and missed a lot of the features in deCONZ so I flipped back and ran into this issue again.  Glad to see there's a workaround, and hopefully a fix eventually.

Leaving this note here in case anyone else runs into a similar issue.
